src/components/GuessBox.js

```diff
@@ -0,0 +1,53 @@
+//This is the options box that will pop up when you hit guess.... ADD CANCEL BUTTON IN HERE
```

Comment:
This component seems to be moving in the right direction. Your CSS is off on it so it's not causing it to sit on the top of the page as you would want your modal to. Look into your components positioning and it's z-index level and a few other things to adjust that to make it sit on top of the page.

Your functionality is moving in the right direction though. You have it returning on the page the one that's selected. You just need to compare that to the location you get back from your fetch from nominatom to see if it was the right place